### DISCUSSION TITLE
Manage sdcard content better

### DISCUSSION THREAD

**Discussion by lshems** [EdgeTX/edgetx#187] [Ideas]
Sdcards are HUGE nowadays, so let's not optimise storage capacity. No need for that.

That allows to have the radio use a directory as curdir, instead of the root. Than we can switch between firmware versions on the radio, by keeping different folders for different firmware versions.

This also allows for non_version specific folders for non_version related content, such as named voice files, images, log files and model note files.

Finally, this would also allow for radio type specific folders, so we would have one sdcard content set only for all radios, that is automatically supporting backwards compatibility by just including the folder structure and content for versions over time.

Also, predefined templates to be used for easily restoring model files adapted for radio specific configurations would become possible.

One to rule them all...

**Comment by lshems** [comment]
P.S. I would be happy to put some time into this, organising files, thinking about a structure. I just cant program.

So vote, or leave for later ;)

**Comment by pfeerick** [comment]
Do you mind if I convert this to a discussion?

**Comment by lshems** [comment]
Perfect

**Comment by JimB40** [comment]
That's very good concept. Updating SD content with new firmware version was always troublesome.
Can you propose new folder structure, having in mind existing script compatibility where possible?

**Reply by lshems** [reply]
I will be working this evening on some issues.
